other/Dockerfile

```diff
@@ -29,6 +29,14 @@ RUN npm prune --omit=dev
 # Build the app
 FROM base as build
 
+# vars needed for the build (in vite.config)
+#ARG COMMIT_SHA
+#ENV COMMIT_SHA=$COMMIT_SHA
+#ARG SENTRY_AUTH_TOKEN
```

Comment:
Do not pass secrets as build args in docker, this is bad practice.

We want to use a docker build secret instead.

We'd need to do something like this at the build step 👇
```
RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN sh -c 'export SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && npm run build'
```

Reply:
We can pass this secret in the github action 

```yaml
      - name: 🚀 Deploy Production
        if: ${{ github.ref == 'refs/heads/main' }}
        run: |
          flyctl deploy --remote-only \
           --build-arg COMMIT_SHA=${{ github.sha }} 
           --build-secret SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }}
```

Reply:
thanks! done!

Reply:
beautiful! 🫶